nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -235,6 +235,11 @@ language governing permissions and limitations under the License. -->
             <artifactId>json-utils</artifactId>
             <version>0.0.20</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.calcite</groupId>
+            <artifactId>calcite-example-csv</artifactId>
```

Comment:
This seems to be bringing in OpenCSV as a transitive dependency. Between https://commons.apache.org/proper/commons-csv/ and OpenCSV, wouldn't the former be more preferable? Most other contributions seem to be using the commons-csv, few more features, etc. We don't want multiple CSV libraries, do we?